org.gajim.Gajim.Plugin.url_image_preview.metainfo.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>org.gajim.Gajim.Plugin.url_image_preview</id>
+  <extends>org.gajim.Gajim.desktop</extends>
+  <name>URL Image Preview Plugin</name>
```

Comment:
It should probably contain project name here.

Reply:
I don't see why it should contain the project name. Plugins are only listed as part of the project page (at least in gnome-software).

The other plugins available via flathub (https://github.com/flathub/de.wolfvollprecht.UberWriter.Plugin.TexLive and https://github.com/flathub/io.github.Hexchat.Plugin.Perl) also don't contain the project name there.

Reply:
Ah, fair point. Thought it also shows up in search results.